assets/js/components/activation/activation-app.js

```diff
@@ -58,70 +56,56 @@ export function ActivationApp() {
 	const canViewDashboard = useSelect( ( select ) =>
 		select( CORE_USER ).hasCapability( PERMISSION_VIEW_DASHBOARD )
 	);
-	const isConnected = useSelect( ( select ) =>
-		select( CORE_SITE ).isConnected()
-	);
-	const isUsingProxy = useSelect( ( select ) =>
-		select( CORE_SITE ).isUsingProxy()
-	);
 
 	useMount( () => {
 		trackEvent( VIEW_CONTEXT_ACTIVATION, 'view_notification' );
 	} );
 
-	let buttonURL = proxySetupURL || splashURL;
-	let buttonLabel = __( 'Start setup', 'google-site-kit' );
-
-	if ( canViewDashboard ) {
-		buttonURL = dashboardURL;
-		buttonLabel = __( 'Go to Dashboard', 'google-site-kit' );
-	}
+	const buttonURL = canViewDashboard ? dashboardURL : splashURL;
+	const buttonLabel = canViewDashboard
+		? __( 'Go to Dashboard', 'google-site-kit' )
+		: __( 'Start setup', 'google-site-kit' );
 
 	const onButtonClick = useCallback(
 		async ( event ) => {
 			event.preventDefault();
-			await trackEvent( VIEW_CONTEXT_ACTIVATION, 'confirm_notification' );
-
-			if ( ! canViewDashboard && proxySetupURL && ! isConnected ) {
-				await trackEvent(
-					VIEW_CONTEXT_ACTIVATION,
-					'start_site_setup',
-					isUsingProxy ? 'proxy' : 'custom-oauth'
-				);
-			}
-
-			if ( ! canViewDashboard && proxySetupURL ) {
-				await trackEvent(
-					VIEW_CONTEXT_ACTIVATION,
-					'start_user_setup',
-					isUsingProxy ? 'proxy' : 'custom-oauth'
-				);
-			}
+			const eventLabel = canViewDashboard ? 'dashboard' : 'splash';
+			await trackEvent(
+				VIEW_CONTEXT_ACTIVATION,
+				'confirm_notification',
+				eventLabel
+			);
 
 			navigateTo( buttonURL );
 		},
-		[
-			canViewDashboard,
-			proxySetupURL,
-			isConnected,
-			navigateTo,
-			buttonURL,
-			isUsingProxy,
-		]
+		[ buttonURL, canViewDashboard, navigateTo ]
 	);
 
 	if ( ! buttonURL ) {
 		return null;
 	}
 
 	return (
-		<Fragment>
-			<NotificationCounter />
-			<ActivationMain
-				buttonURL={ buttonURL }
-				buttonLabel={ buttonLabel }
-				onButtonClick={ onButtonClick }
-			/>
-		</Fragment>
+		<Grid>
+			<Row>
+				<Cell size={ 12 }>
+					<Logo />
+					<h3 className="googlesitekit-heading-3 googlesitekit-activation__title">
+						{ __(
+							'Congratulations, the Site Kit plugin is now activated.',
+							'google-site-kit'
+						) }
+					</h3>
+					<Button
+						id="start-setup-link"
+						className="googlesitekit-start-setup"
+						href={ buttonURL }
```

Comment:
I think this `href` attribute is redundant since we call the `navigateTo` function in the `onButtonClick` callback, right?

Reply:
Yes works fine without it. I have removed it.